project-base/storefront/components/Basic/Flag/Flag.tsx

```diff
@@ -1,4 +1,4 @@
-import { twMergeCustom } from 'utils/twMerge';
+import { twMergeCustom } from 'helpers/twMerge';
```

Comment:
Now when I am seeing this, I wonder if it would be better rather use `utils` folder instead `helpers`. Since `utils` seems more generic where we can put everything, not just `helpers`, but vice versa seems to me like not every utility which we need is considered as helper. Though `helpers` and `utils` are almost synonyms. But just something to thing about twice to be sure this is correct step.

Reply:
I think this is an unnecessary change which will only cause conflicts. If we decide to do it after all, we can do it separately